mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -9,6 +9,7 @@
 from copy import deepcopy
 
 import numpy as np
+from numpy.core.records import fromarrays
```

Comment:
IIRC no.core is semi private, is there an explicitly public function you could use?

Reply:
I changed it to `np.rec.fromarrays` instead.